script.js

```diff
@@ -358,3 +357,35 @@ Object.keys(navLinks).forEach((k) => {
     navLinks[k].addEventListener('click', closeMobileMenu);
   }
 });
+
+// Form Validation
+const formOne = document.getElementById('formOne');
+const email = document.getElementById('email');
+let errorMsg = document.getElementById('errorMsg');
+let emailTest = '';
+let formTest = false;
+
+// eslint-disable-next-line no-trailing-spaces
+/* Get lowercase of each vale in 
+email for testing real user case value */
+email.addEventListener('keyup', () => {
+  emailTest = email.value.toLowerCase();
+});
+
+formOne.addEventListener('submit', (e) => {
+  if (emailTest !== email.value) {
+    errorMsg.innerText = 'Email should be lowercase.';
+    if (formTest === false) {
+      errorMsg.classList.toggle('hideErr');
+    }
+    formTest = true;
+    // To remove the errMsg when email input is clicked
+    email.addEventListener('click', () => {
+      if (formTest === true) {
+        errorMsg.classList.toggle('hideErr');
```

Comment:
## [OPTINAL]

- [ ]  To improve the design I would suggest you add some styling error interaction on the input that contains the validation error. Just consider it as a challenge that can improve your UI.  @Ginohmk  [see the challenge here](https://andersonlebon.github.io/Portfolio-w2/). 
![image](https://user-images.githubusercontent.com/65068771/146368315-34dae144-e2d0-4e7a-a7b4-6061e59892b0.png)
![image](https://user-images.githubusercontent.com/65068771/146368744-375816e7-f4d1-42d1-875b-ae22fa593453.png)

Reply:
THanks

Reply:
I would also love to challenge you about your local storage, when someone deletes one field and refreshes the page, the deleted data should not come back. 😄

Reply:
![image](https://user-images.githubusercontent.com/65068771/146374540-511a86a9-9373-46d7-910f-69197ca47d2e.png)